### DISCUSSION TITLE
WhatsApp server returns abnormal data，why?

### DISCUSSION THREAD

**Discussion by RamboNie** [tulir/whatsmeow#531] [WhatsApp protocol Q&A]
Some WhatsApp accounts are successfully logged in. After about 1 minute, the WhatsApp server will return byte[] [-8, 1, 2], and then the parsing exception will occur and the connection will be disconnected. Can anyone help confirm this? Thanks very much

**Comment by rs2152** [comment]
I'm facing connectivity issue,
In local machine it is working fine, but when deployed on ec2 server.
Getting streamerror. Not able to send any message 
Any suggestions?
